lib/core/log/cne_log.h

```diff
@@ -36,9 +36,26 @@ enum {
     CNE_LOG_NOTICE,    /**< Normal but significant condition. */
     CNE_LOG_INFO,      /**< Informational.                    */
     CNE_LOG_DEBUG,     /**< Debug-level messages.             */
-    CNE_LAST_LOG
+    CNE_LOG_LAST
 };
 
+#define foreach_cndp_log_level \
```

Comment:
we could consolidate into 1 set of log levels?

Reply:
ok